extensions/BugModal/web/bug_modal.js

```diff
@@ -868,12 +868,20 @@ $(function() {
             if ($('#comment').val() != reply_text) {
                 $('#comment').val($('#comment').val() + reply_text);
             }
-            autosize.update($('#comment'));
-            $.scrollTo($('#comment'), function() { $('#comment').focus(); });
-        });
-
-    // auto-enlarge comment area (up to its max-height)
-    autosize($('#comment'));
+            
+            // calling autosize.update or autosize
+            if (BUGZILLA.user.settings.autosize_comment) {
+                autosize.update($('#comment'));
+            }
```

Comment:
structure got all messed up here.
```javascript
if (BUGZILLA.user.settings.autosize_comment) {
    autosize.update($('#comment'));
}
```
needs to happen inside the un-named function/callback ```.click(function(event) {``` which begins on line 846. After that, there needs to be a }) that closes off that function.

Then the below code happens -- which will then be one indentation level lower.